mlflow/__init__.py

```diff
@@ -247,3 +249,15 @@
     from mlflow import gateway  # noqa: F401
 
     __all__.append("gateway")
+
+
+def _configure_mlflow_loggers():
```

Comment:
I'm not sure if this function is necessary. I think we only need to change the logging level in https://github.com/mlflow/mlflow/blob/master/mlflow/utils/logging_utils.py#L82C20-L82C37, no? I think @B-Step62  should have a better knowledge of the project than me.

Reply:
@B-Step62 , any suggestion here?

Reply:
I agree with @Flametaa here. There should not be multiple ways to configure logging.

Reply:
I agree with @Flametaa here. There should not be multiple ways to configure logging. Can we remove the new function and document how to change logger level using the env var?